apps/dapp-console/app/mixpanel/mixpanel.tsx

```diff
@@ -0,0 +1,14 @@
+import mixpanelBrowser from 'mixpanel-browser'
+
+const initMixpanel = () => {
+  if (
```

Comment:
would just do `if(!process.env.VITE_MIXPANEL_TOKEN)` then we can have the system setup so if it doesn't exist we just don't init mixpanel, we can ignore passing in incorrect strings